Software/Development/SemesterProject/App/analogAcquisitionManager.c

```diff
@@ -359,9 +359,9 @@ void createAnalogSignals(void)
      * Then pass the maximum and then the minimum value of the threshold you want to set.
      */
     setCurrentThresholds(&currentThreshold[0], 250, -250);
-    setDCLinkVoltageThresholds(&DCLinkVoltageThreshold[0], 45, 25);
+    setDCLinkVoltageThresholds(&DCLinkVoltageThreshold[0], 45, 0);
     setControlSupplyVoltageThresholds(&controlVoltageThreshold[0], 30, 15);
-    setThermometerThresholds(&thermalThreshold[0], 100, 10);
+    setThermometerThresholds(&thermalThreshold[0], 200, 10);
```

Comment:
Why 200 ?¿ It seems to high, mosfets would break long before...

Reply:
The 200 deg is to not have an error while the sensors are not in use

Reply:
Sounds fine!